e2eshark/run.py

```diff
@@ -843,7 +843,7 @@ def runTestUsingClassicalFlow(args_tuple):
     if args.verbose:
         print(f"Running classical flow for test {testName}")
     ireeInputType = "onnx"
-    if SHARED_TORCH_MLIR_BUILD:
+    if SHARED_TORCH_MLIR_BUILD or mode=="turbine":
         ireeInputType = "torch"
```

Comment:
You can also omit `--iree-input-type` and let the auto pipeline detect the input type based on the input program. I recommend that, especially since there is code like this that tries to be explicit but then gets the type wrong...

Reply:
Oh nice! Yeah that's a lot better